wrappers/services.go

```diff
@@ -492,18 +492,19 @@ WantedBy={{.ServicesTarget}}
 
 func genServiceSocketFile(appInfo *snap.AppInfo, socketName string) []byte {
 	socketTemplate := `[Unit]
-# Auto-generated, DO NO EDIT
+# Auto-generated, DO NOT EDIT
 Description=Socket {{.SocketName}} for snap application {{.App.Snap.InstanceName}}.{{.App.Name}}
 Requires={{.MountUnit}}
-Wants={{.PrerequisiteTarget}}
-After={{.MountUnit}} {{.PrerequisiteTarget}}
```

Comment:
With that removed, we can stop passing `		PrerequisiteTarget string` into the template in genServiceSocketFile, right?